lib/BuildSystem/SwiftTools.cpp

```diff
@@ -220,6 +223,11 @@ class SwiftCompilerShellCommand : public ExternalCommand {
     if (isLibrary) {
       result.push_back("-parse-as-library");
     }
+    if (enableWholeModuleOptimization) {
+      result.push_back("-whole-module-optimization");
+      result.push_back("-num-threads");
+      result.push_back("1");
```

Comment:
atbuild would be interested in setting a value for num-threads, so that's a datapoint.